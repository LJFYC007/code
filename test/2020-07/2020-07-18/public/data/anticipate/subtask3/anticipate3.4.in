3
50 -1000000 1000000
-971964 -958766 -955032 -927896 -898814 -865974 -850033 -826362 -804048 -723664 -616220 -603912 -558955 -539191 -538422 -499670 -498015 -462201 -459037 -450190 -437121 -416401 -380242 -353623 -344579 -260663 -247613 -184120 -130399 -5169 33957 251140 266694 289863 420852 447511 513840 549026 554285 560510 584225 598398 639100 724700 767822 852151 855090 902402 925348 996594 
500920 500448 500631 500193 500697 500484 500299 500844 500489 500996 500770 500929 500849 500962 500141 500916 500691 500801 500722 500026 500615 500693 500179 500336 500394 500102 500524 500897 500130 500378 500689 500973 500227 500048 500463 500997 500837 500973 500462 500152 500459 500315 500237 500361 500173 500757 500150 500102 500183 500252 
