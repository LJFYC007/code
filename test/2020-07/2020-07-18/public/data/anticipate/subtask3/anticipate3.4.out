0.014114425080
0.006539284613
0.002144495332
0.013776751637
0.014867932405
0.016114403211
0.008304971336
0.011950683512
0.014241598583
0.043972405618
0.047038789831
0.009538971108
0.020108862437
0.007852402793
0.004712049675
0.015115289154
0.005165002841
0.013775039526
0.002419108612
0.004901393035
0.007723856027
0.012849180259
0.016814932665
0.010608380637
0.017446858606
0.029949741396
0.016435332770
0.030813731328
0.044515326699
0.043489761281
0.072617119252
0.050642681592
0.010230079587
0.049292974467
0.029885178982
0.028628419024
0.021865310013
0.006136670589
0.002996228941
0.009996461766
0.008185480156
0.018029789484
0.040379821214
0.025175717147
0.041436686467
0.004659239210
0.022560988254
0.012219491774
0.035994730106
0.001765969966
